src/app/client/src/app/modules/public/module/signup/components/signup/signup.component.ts

```diff
@@ -58,6 +59,7 @@ export class SignupComponent implements OnInit, OnDestroy, AfterViewInit {
     this.tenantDataSubscription = this.tenantService.tenantData$.subscribe(
       data => {
         if (data && !data.err) {
+          this.secondaryLogo = data.tenantData.secondaryLogo;
```

Comment:
@sohailamjad12 
- direct reference of object is not recommended
- add Optional chaining to avoid `cannot read of undefined exception`
CC: @mangeshmane @rajeevsathish